docs/src/drivers/hostmot2.adoc

```diff
@@ -633,107 +633,91 @@ component. Each pwmgen instance has the following pins and parameters:
   boards may have different clocks, resulting in different max PWM
   frequencies. If the user attempts to set the frequency too high, it
   will be clipped to the max supported frequency of the board.
-  Frequencies below about 5&thinsp;Hz are not terribly accurate, but above 5&thinsp;Hz
-  they're pretty close.
+  Frequencies below about 5&thinsp;Hz are not terribly accurate,
+  but above 5&thinsp;Hz they are pretty close.
 
 === Output Parameters
 
 The output pins of each PWMGen have two additional parameters. To find
-which I/O pin belongs to which output run 'dmesg' as described above.
+which I/O pin belongs to which output run `dmesg` as described above.
 
-* 'invert_output' - (Bit, RW) This parameter only has an effect if the
-  'is_output' parameter is true. If this parameter is true, the output
-  value of the GPIO will be the inverse of the value on the 'out' HAL
-  pin.
+* `invert_output` - (Bit, RW) This parameter only has an effect if the
+  `is_output` parameter is true. If this parameter is true, the output
+  value of the GPIO will be the inverse of the value on the `out` HAL pin.
 
-* 'is_opendrain' - (Bit, RW) If this parameter is false, the GPIO behaves
+* `is_opendrain` - (Bit, RW) If this parameter is false, the GPIO behaves
   as a normal output pin: the I/O pin on the connector is driven to the
-  value specified by the 'out' HAL pin (possibly inverted).
+  value specified by the `out` HAL pin (possibly inverted).
   If this parameter is true, the GPIO behaves as an open-drain pin.
-  Writing 0 to the 'out' HAL pin drives the I/O pin low, writing 1 to
-  the 'out' HAL pin puts the I/O pin in a high-impedance state. In this
+  Writing 0 to the `out` HAL pin drives the I/O pin low, writing 1 to
+  the `out` HAL pin puts the I/O pin in a high-impedance state. In this
   high-impedance state the I/O pin floats (weakly pulled high), and other
   devices can drive the value; the resulting value on the I/O pin is
-  available on the 'in' and 'in_not' pins.
+  available on the `in` and `in_not` pins.
   Only full GPIO pins and I/O pins used as outputs by active module
   instances have this parameter.
 
 [[sec:hm2-encoder]]
 == Encoder
 
 Encoders have names like
-'hm2_<BoardType>.<BoardNum>.encoder.<Instance>.'. 'Instance' is a
-two-digit number that corresponds to the HostMot2 encoder instance
-number. There are 'num_encoders' instances, starting with 00.
+`hm2__<BoardType>__.__<BoardNum>__.encoder.__<Instance>__.`.
```

Comment:
This renders not correctly

Reply:
I need a better previewer - or the transition just needs to happen.

Reply:
Have undone the eye candy.